src/journal/views.py

```diff
@@ -1288,6 +1288,21 @@ def editorial_team(request, group_id=None):
 
     return render(request, template, context)
 
+@has_journal
+def author_list(request):
+    """
+    Displays list of authors. 
```

Comment:
There is a trailing whitespace in this line

Reply:
I've updated these to remove the whitespaces.
Do you guys have a styleguide or a link to one that you are following? 
The spacing is a little inconsistent throughout.

Reply:
We're trying to have new code stick to the PEP8 style guide. Our old code will be brought in line with it asap :+1: